bin/publisher/agents_whitelist.json

```
@@ -3,6 +3,9 @@
     {
       "name": "wayback-machine-agent"
     },
+    {
+      "name": "document-explorer"
```

This I think requires DI so Studio version limits this from showing up on 1.3, 1.4 that just do not work with these
```suggestion
      "name": "document-explorer"
      "required_studio_version": ">=1.5.0"
```